workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/CodeWriter/Scopes/Abstract/ScopeBody.cs

```diff
@@ -123,14 +123,9 @@ public void Return(string toReturn)
             Line($"return {toReturn};");
         }
 
-        public void ProfilerStart(string name)
+        public void ProfileScope(string markerVariableName, Action<CustomScopeBlock> populate)
```

Comment:
:rip: @paulbalaji's docs

Reply:
it's fine - only a few LoC docs change required